CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+## Unreleased - as of Sprint 53 end 2017-01-30
```

Comment:
@bdunne is there a way to tell which PRs were incuded?  Is it as simple as date?